CMakeLists.txt

```diff
@@ -181,26 +182,33 @@ else()
         # generic, which isn't very good in some cases. march=native looks at
         # cpuid info and then chooses the best microarch it can (and replaces
         # the flag), so use that for tune.
+        
+        if (CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64|i386")
```

Comment:
I just wanna sure does it include all PROCESSOR flags which we want to cover? i386|i686|amd64|x86_64|AMD64

See ref in [1]

[1] https://github.com/ceph/ceph/blob/master/cmake/modules/SIMDExt.cmake#L48